code/datums/traits/negative.dm

```diff
@@ -225,6 +225,20 @@
 	lose_text = "<span class='notice'>You feel awake again.</span>"
 	medical_record_text = "Patient has abnormal sleep study results and is difficult to wake up."
 
+/datum/quirk/clownphobia
+	name = "Clownphobia"
+	desc = "You fear clowns with your life"
+	value = -1
+	mob_trait = TRAIT_CLOWNPHOBIA
+	gain_text = "<span class='danger'>Clowns starts to get you on your nerves.</span>"
+	lose_text = "<span claass='notice'>Clowns no longer get on your nerves.</span>"
+	medical_record_text = "Patient has developed a phobia towards clowns."
+
+/datum/quirk/clownphobia/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.gain_trauma(/datum/brain_trauma/mild/phobia, TRUE, "clowns")
```

Comment:
This gets cured by brain surgery.

Reply:
Getting a lobotomy to cure your fear of clowns sounds hilarious tbh

Reply:
It does, but normal brain surgery also cures it. Requires TRAUMA_RESILIENCE_LOBOTOMY